rskj-core/src/main/java/org/ethereum/net/server/ChannelManagerImpl.java

```diff
@@ -92,8 +92,8 @@ public void init() {
         mainWorker.scheduleWithFixedDelay((Runnable) () -> {
             try {
                 processNewPeers();
-            } catch (Throwable t) {
-                logger.error("Error", t);
+            } catch (Exception e) {
```

Comment:
Are we sure about this ? Any reasoning ?

Reply:
I'll revert this change until a better check is done about this